controllers/dspipeline_params.go

```diff
@@ -630,6 +632,57 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
+		if cfg := p.APIServer.CustomKfpLauncherConfig; cfg != nil && cfg.Name != "" {
+			if cm, err := util.GetConfigMap(ctx, cfg.Name, p.Namespace, client); err != nil {
+				// If the custom kfp-launcher configmap is not available, that is OK
+				if !apierrs.IsNotFound(err) {
+					log.Info(fmt.Sprintf("Encountered error when attempting to fetch ConfigMap: [%s], Error: %v", cfg.Name, err))
+					return err
+				}
+			} else {
+				// Consume all the required information.
+				configMapName := config.CustomKfpLauncherConfigMapName
+				dataValues := util.GetConfigMapValues(cm)
+
+				for _, val := range dataValues {
+					if val != "" {
+						p.DataValue = append(p.DataValue, []byte(val))
+					}
+				}
```

Comment:
you don't need to do this, `util.GetConfigMapValues(cm)` already returns a string, which you can use for `StringData` directly instead of using `Data` in line 669

Reply:
As that was returning a slice of strings. They needed to be appended and couldn't have been directly passed as a string.